sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -237,29 +237,28 @@ for page in result.pages:
             words = get_words(page, line)
             print(
                 f"...Line # {line_idx} has word count {len(words)} and text '{line.content}' "
-                f"within bounding polygon '{line.polygon}'"
+                f"within bounding polygon '{format_polygon(line.polygon)}'"
             )
 
     if page.words:
         for word in page.words:
-            print(
-                f"......Word '{word.content}' has a confidence of {word.confidence}"
-            )
+            print(f"......Word '{word.content}' has a confidence of {word.confidence}")
 
     if page.selection_marks:
         for selection_mark in page.selection_marks:
             print(
                 f"Selection mark is '{selection_mark.state}' within bounding polygon "
-                f"'{selection_mark.polygon}' and has a confidence of {selection_mark.confidence}"
+                f"'{format_polygon(selection_mark.polygon)}' and has a confidence of {selection_mark.confidence}"
             )
 
 if result.paragraphs:
     print(f"----Detected #{len(result.paragraphs)} paragraphs in the document----")
     for paragraph in result.paragraphs:
-        print(f"Found paragraph with role: '{paragraph.role}' within {paragraph.bounding_regions} bounding region")
+        print(
+            f"Found paragraph with role: '{paragraph.role}' within {format_bounding_region(paragraph.bounding_regions)} bounding region"
+        )
         print(f"...with content: '{paragraph.content}'")
 
-    result.paragraphs.sort(key=lambda p: (p.spans.sort(key=lambda s: s.offset), p.spans[0].offset))
     print("-----Print sorted paragraphs-----")
     for idx, paragraph in enumerate(result.paragraphs):
```

Comment:
Why do we split paragraph info into 2 loops instead of outputting the role/boundingRegions together with spans?

Reply:
Merged the two loops.